lib/linguist/heuristics.rb

```diff
@@ -61,9 +61,6 @@ def call(data)
       @heuristic.call(data)
     end
 
-    # Common heuristics
-    ObjectiveCRegex = /^[ \t]*@(interface|class|protocol|property|end|synchronised|selector|implementation)\b/
```

Comment:
I'm not sure I understand the rationale for having a different heuristic for `Objective-C` in different scenarios. Is this necessary? There are good reasons for wanting to extract this into a variable.